pkg/safeguards/constraints/ContainerResourceLimits/deployments/CRL_Error_Manifest.yaml

```diff
@@ -17,5 +13,9 @@ spec:
         app: my-app
     spec:
       containers:
-        - name: my-container
-          image: nginx:latest
\ No newline at end of file
+        - name: badcontainer
+          image: badimage
+          resources:
```

Comment:
both of these files have resource limits specified. What is the error you're expecting to receive here?